llmfoundry/models/hf/hf_causal_lm.py

```diff
@@ -393,12 +361,9 @@ def _autoset_attn_implementation_monkeypatch(
                 pretrained_lora_id_or_path,
             )
 
-        if prepare_for_fsdp:
```

Comment:
can we leave this arg here? It is part of this public api and there isn't really any reason to remove it.

Reply:
do you mean we should pass `prepare_for_fsdp=False` to `build_inner_model`?

Reply:
yea